.circleci/config.yml

```diff
@@ -1763,6 +1763,8 @@ workflows:
           skip_slow_tests: true
           notify: true
       - cannon-build-test-vectors
+      - todo-issues:
+          name: todo-issues-check
```

Comment:
Oooo, this is not a job that's designed to run pre-merge.  It _will_ cause a heap of issues and start failing on every PR because someone happened to close an issue.  It was run on a schedule with slack notifications very very deliberately.

We should revert this.